lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Very simple lint driver. This is primarily because shellcheck needs to
+# be told the files to check as arguments.
+
+set -e
+
+PROG=$(basename "$0")
+PROGDIR=$(dirname "$0")
+
+cd "$PROGDIR"
+flake8
```

Comment:
For GLib we went with [`flake8`](https://gitlab.gnome.org/GNOME/glib/-/blob/main/.gitlab-ci/run-flake8.sh) *and* [`black`](https://gitlab.gnome.org/GNOME/glib/-/blob/main/.gitlab-ci/run-black.sh), because the latter does a better job of formatting things. But I’m probably bikeshedding at this point. Ignore me.

Reply:
I'm not always happy with `black`'s opinions, but it's probably the way to go. For now I'm going to punt since that's not currently part of even the unenforced flow.